### PR TITLE
Insure that headers are correctly cased before signature verification…

### PR DESCRIPTION
**Pull request by OGKevin**
Closes bunq/sdk_csharp#49

### CODE REVIEW

File: BunqSdk/Security/SecurityUtils.cs [hunk 1]
```diff
@@ -127,6 +133,20 @@ private static string GenerateRequestHeadersSortedString(HttpRequestMessage requ
             );
         }
 
+        private static string GetHeaderNameCorrectyCased(string headerName)
```
Comment (epels) [inline BunqSdk/Security/SecurityUtils.cs]: GetHeaderName**Correcty**Cased

Comment (epels) [inline BunqSdk/Security/SecurityUtils.cs]: Test?

File: BunqSdk/Security/SecurityUtils.cs [hunk 2]
```diff
@@ -127,6 +133,20 @@ private static string GenerateRequestHeadersSortedString(HttpRequestMessage requ
...
+        {
+            headerName = headerName.ToLower();
+            headerName = headerName.First().ToString().ToUpper() + headerName.Substring(1);
```
Comment (epels) [inline BunqSdk/Security/SecurityUtils.cs]: https://msdn.microsoft.com/nl-nl/library/system.globalization.textinfo.totitlecase(v=vs.110).aspx ?

File: BunqSdk/Security/SecurityUtils.cs (hunk 1 above)
Comment (andrederoos) [inline BunqSdk/Security/SecurityUtils.cs]: `GetHeaderNameCorrectlyCased`

File: BunqSdk/Security/SecurityUtils.cs (hunk 2 above)
Comment (andrederoos) [reply BunqSdk/Security/SecurityUtils.cs]: what emile says ;-)

Comment (OGKevin) [reply BunqSdk/Security/SecurityUtils.cs]: Regex is already used in the other SDK's. So for consistency i would like to keep using regex.

File: BunqSdk/Security/SecurityUtils.cs (hunk 1 above)
Comment (OGKevin) [reply BunqSdk/Security/SecurityUtils.cs]: Lets create a follow up issue to add tests in all SDK's. I've done a quick dirty test by manipulating the headers right after the response is received to see if it indeed works, which it does. But a test unit is indeed needed to ensure that in the future it does not break.

File: BunqSdk/Security/SecurityUtils.cs (hunk 2 above)
Comment (epels) [reply BunqSdk/Security/SecurityUtils.cs]: @OGKevin This is what I meant: https://dotnetfiddle.net/QdElDN

File: BunqSdk/Security/SecurityUtils.cs (hunk 1 above)
Comment (OGKevin) [reply BunqSdk/Security/SecurityUtils.cs]: bunq/sdk_csharp#60

File: BunqSdk/Security/SecurityUtils.cs (hunk 2 above)
Comment (OGKevin) [reply BunqSdk/Security/SecurityUtils.cs]: Yes I understand. But for the sake of maintaining, I prefer consistency over all the SDK's. So lets keep the SDK's as consistent with each other as we can.

**Pull request merged**